setup.py

```diff
@@ -102,6 +102,7 @@ def read(*names, **kwargs):
         "appdirs",
     ],
     extras_require={
+        "dbt": ["dbt>=0.18.1"],

```

Comment:
Would be curious to see how this works with earlier versions of `dbt`. If someone hasn't upgraded to the latest version (because of breaking changes/whatnot) then they wouldn't be able to take advantage of this new `sqlfluff` feature.

Reply:
I agree, but I tried on `0.17.2` and found that some methods I depend on (from the DBT package) had moved places.

Since the classes we use from the DBT project aren't an "official API", or something they expect users to call directly, we can't really expect them to be stable.

Should we agree on a subset of DBT versions to support, and have different tox environments for each DBT version?

We can start with 0.17 and 0.18

Reply:
Ah didn't realize that the function moved on you 🚚 

How tedious do you would it be to support multiple versions? If it's a pain, then maybe we should keep this requirement in there.

Reply:
I changed the code yesterday to support both versions, wasn't too much of a pain :)